src/frontend-samples/reality-data-sample/sampleSpec.ts

```diff
@@ -12,12 +12,13 @@ export function getRealityDataSpec(): SampleSpec {
     label: "Reality Data",
     image: "reality-data-thumbnail.png",
     iTwinViewerReady: true,
-    readme: async () => import("!!raw-loader!./readme.md"),
+    readme: async () => import("!raw-loader!./readme.md"),
+    walkthrough: async () => import("!annotation-loader!./annotations.md"),
```

Comment:
This will create a json object array from this file that can be consumed by the annotations module